src/main/resources/hudson/plugins/active_directory/ActiveDirectorySecurityRealm/help-userFromHTTPHeader.html

```diff
@@ -0,0 +1,8 @@
+<div>
+    Authenticates the user in Jenkins via a HTTP header field.
+    You can disable the behaviour if set this value to "".
+
+    SECURITY WARNING: Please make sure that this header cannot be provided
```

Comment:
You can wrap this in `<div class="warning">` or `<strong class="warning-inline">` or similar to make it stand out better.